web_responsive/static/src/xml/apps.xml

```diff
@@ -14,9 +14,9 @@
         <t t-jquery=".o_app &gt; t" t-operation="replace">
             <t t-call="web_responsive.AppIcon" />
         </t>
-        <!-- Same hotkey as in EE -->
+        <!-- Same hotkey as in EE (Alt + Shift + A doesn't work on Chrome) -->
         <t t-jquery=".full" t-operation="attributes">
-            <attribute name="accesskey">a</attribute>
+            <attribute name="accesskey">h</attribute>
```

Comment:
This is a breaking change, so unless you provide both keys, I don't want to change it.

Reply:
I added some hack to support both keys. It's quite dirty but works. Tested on Chrome and Edge on Windows.